README.md

```diff
@@ -1,3 +1,34 @@
 # Futbol
 
 Starter repository for the [Turing School](https://turing.io/) Futbol project.
+
+Built with Ruby 2.7.2, CSV, Rspec 3.12.0
+
+
+**We plan to meet daily at 2:30pm MST for stand-ups and work together or be available for most of the afternoon and evening with a dinner break.**
+
+**Our team plans to use _Github Projects_ to organize our tasks/tickets that need to be accomplished in our project. We are also using _Miro_ to organize our thoughts and to mind-map/brainstorm our whole project.  We collectively add inputs when brainstorming and then we can edit tickets in our *Github* projects.** 
+[GitHub Project](https://github.com/users/grace-joh/projects/3)
+[Miro Board](https://miro.com/app/board/uXjVPiopTmI=/)
+
+**To decide on our approach, we looked at a couple ways of organizing our ideas and setting up where we would track everyone's work. We settled on using Miro for planning and brainstorming, and using github projects to organize individual tasks, which we picked over trello because we wanted to try something most of us had not used before.**
+
+## update 
+** **Code design** ^^ **
+
+## Group DTR
+[DTR Document](https://docs.google.com/document/d/1Nqv1R2Ognb3WXtzAy6NgxfQZs2xR7j-oHdXkRTyv0K4/edit?usp=sharing)
+Completed on Feb. 27, 2023
+
+Amy Spears
+[LinkedIn](https://www.linkedin.com/in/amy-marie-spears-900997105/)
+[GitHub](https://github.com/amspears007)
+Grace Joh
+[LinkedIn](http://www.linkedin.com/in/graceehjoh)
+[GitHub](https://github.com/grace-joh)
+Julian Beldotti
+[LinkedIn](https://www.linkedin.com/in/julian-beldotti-1a427824/)
+[GitHub](https://github.com/JCBeldo)
+Logan Cole
+[LinkedIn](https://www.linkedin.com/in/logan-cole-ab2179188/)
+[GitHub](https://github.com/exasperlnc)
```

Comment:
I think it would be helpful to use `-` to make these bullet points with indents so that they are each on a new line. Using the Preview function can help see what the markdown file will look like vs in the text editor.

Reply:
I did update these, but they are not reflected the same way in a Readme as a normal Markdown file is. The links have changed colour, but they don't have bullets.  Or maybe that's just in my version of VS code. I have a different skin than default.  Let me know if you see a diff when you pull?